### DISCUSSION TITLE
MQTT Integration adapted for Homie convention

### DISCUSSION THREAD

**Discussion by mwildbolz** [john30/ebusd#567] [Q&A]
I really like the new feature of the integration layer on top of the MQTT protocol with its fearures for auto discovery.

@john30 Do you know the [Homie MQTT convention ](https://homieiot.github.io/#)?
Would it be also possible to create a working integration configuration which sticks to this convention?

Advantage would be, that Homie is natively understood by openHAB via the [MQTT Homie binding](https://www.openhab.org/addons/bindings/mqtt.homie/) and would therefore allow an easy integration into openHAB...

**Comment by john30** [comment]
would be possible in principle, but I didn't like the approach of having to send an MQTT message for each and every litte bit of attribute. that's why I was focusing on HA first

**Reply by mwildbolz** [reply]
Do I understand you right, that it would be possible to write an integration file/template to also adapt to the Homie specification without the need of reprogramming the ebusd core??

That would be fine...

**Reply by john30** [reply]
no, the code needs to be adjusted to support sending an MQTT topic for each little piece of info
